src/sql/azdata.d.ts

```diff
@@ -3210,6 +3211,11 @@ declare module 'azdata' {
 		DataFit = 2		// columns use sizing based on cell data, horiz scroll bar present if more cells than visible in view area
 	}
 
+	export interface RowDetailsOptions {
```

Comment:
All new things should be added to https://github.com/microsoft/azuredatastudio/blob/main/src/sql/azdata.proposed.d.ts